lib/scala/kg/src/main/scala/io/github/tetherlessworld/mcsapps/lib/kg/stores/postgres/AbstractPostgresKgStore.scala

```diff
@@ -0,0 +1,91 @@
+package io.github.tetherlessworld.mcsapps.lib.kg.stores.postgres
+
+import io.github.tetherlessworld.mcsapps.lib.kg.models.edge.KgEdge
+import io.github.tetherlessworld.mcsapps.lib.kg.models.node.KgNode
+import io.github.tetherlessworld.mcsapps.lib.kg.models.source.KgSource
+import play.api.db.slick.{DatabaseConfigProvider, HasDatabaseConfigProvider}
+import slick.jdbc.PostgresProfile
+
+
+abstract class AbstractPostgresKgStore(protected val dbConfigProvider: DatabaseConfigProvider) extends HasDatabaseConfigProvider[PostgresProfile] {
+  import profile.api._
+
+  lazy val KgNodes = TableQuery[KgNodeTable]
+  lazy val KgEdges = TableQuery[KgEdgeTable]
+  lazy val KgSources = TableQuery[KgSourceTable]
+
+  private class KgNodeTable(tag: Tag) extends Table[KgNode](tag, "KgNodes") {
```

Comment:
Usual SQL convention is singular table and column names, with everything lower snake_case.

Reply:
Ah okay changed